type_var_and_const.go

```diff
@@ -294,8 +298,6 @@ func (p *ValueDecl) endInit(cb *CodeBuilder, arity int) *ValueDecl {
 			}
 		}
 	}
-	cb.stk.PopN(arity)
-	cb.endInitExpr(p.old)
 	if p.at >= 0 {
 		cb.commitStmt(p.at) // to support inline call, we must emitStmt at EndInit stage
```

Comment:
this statement also need move to the `defer statement`?

Reply:
moved